### DISCUSSION TITLE
Winamp skins on Ubuntu (snap)

### DISCUSSION THREAD

**Discussion by LibbyBibby** [audacious-media-player/audacious#1489] [General]
I cannot for the life of me figure out how to access the skins folders, I have people telling me to use the command line with root permissions but why on earth should it be that difficult just to import a skin? if anyone has an easier way to get these things into the skin folder I'd really appreciate some advice

**Comment by mschwendt** [comment]
You can store skins in $HOME/.local/audacious/Skins/ directly as archive files. That follows the XDG Base Directory Specification for $XDG_DATA_HOME.

**Reply by LibbyBibby** [reply]
what if there is no audacious file in my .local? should I just create one myself?

**Reply by mschwendt** [reply]
Of course.

**Reply by Fr-Dae** [reply]
for me , Lubuntu22.04, it's on `/usr/share/audacious/Skins/`
but you need root perm to acces it (i don't know why)
for me, it's `sudo pcmanfm /usr/share/audacious/Skins/`
or `cp <~/folder /usr/share/audacious/Skins/>

but with interface it's better =)

**Reply by mschwendt** [reply]
> for me , Lubuntu22.04, it's on `/usr/share/audacious/Skins/` but you need root perm to acces it (i don't know why) for me,
>

That is because it is the global directory in a system installation path, whereas the earlier replies refer to the local directory below user's home directory.

**Reply by Fr-Dae** [reply]
> > for me , Lubuntu22.04, it's on `/usr/share/audacious/Skins/` but you need root perm to acces it (i don't know why) for me,
> 
> That is because it is the global directory in a system installation path, whereas the earlier replies refer to the local directory below user's home directory.

except that the directory you're talking about doesn't exist on ubuntu and Lubuntu

**Reply by mschwendt** [reply]
> except that the directory you're talking about doesn't exist on ubuntu and Lubuntu

That's normal and expected. But you can create it yourself. Many other directories don't exist until something/somebody stores files in them. Empty directories would not serve any purpose.

**Reply by Fr-Dae** [reply]
why should i create a folder ? audacious could use it ?

**Reply by mschwendt** [reply]
> why should i create a folder ? audacious could use it ?

Please read the earlier replies in this topic.

Audacious supports the directory as per the XDG Base Directory Specification for $XDG_DATA_HOME. But if it doesn't exist because YOU don't use it for storing skins, for example, it makes no sense to create empty directories.

If you download skins from the Internet and install them yourself, store them in your user's local directory. Not in the system installation path where you need superuser root privileges to do that.

**Comment by Fr-Dae** [comment]
btw, if you can share your creation, i'm curious
